basiclibrary/src/main/java/Library.java

```diff
@@ -79,10 +80,45 @@ public static void calculateAverages(int[][] monthTempsByWeek) {
         return highLow;
     }
 
-    public static String tally() {
+    public static String tally(ArrayList votes) {
```

Comment:
Dave, Please revisit the parameter you take in here it has no generic defining - also is it an ArrayList or a List?